templates/centos7/common/base.dockerfile.j2

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2019-2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+FROM centos:7
+
+LABEL Description="This is the {{ distribution }} image for {{ product_name }} on Centos 7"
+LABEL Vendor="Intel Corporation"
+
+USER root
+WORKDIR /
+
+SHELL ["/bin/bash", "-xo", "pipefail", "-c"]
+
+ARG DEPENDENCIES="autoconf \
+                  automake \
+                  gcc-c++ \
+                  make \
+                  gtk2 \
+                  gobject-introspection \
+                  glib2 \
```

Comment:
remove the following components, OpenCV team said it doesn't need.                  
 gobject-introspection \
                  glib2 \
                  gdk-pixbuf2 \